### PR TITLE
Event Locales & Yacht Heist Alert

### PR DESCRIPTION
**Pull request by NoobySloth**
Added English locales for cl_events and cl_extraalerts

### CODE REVIEW

**Comment by NoobySloth** [comment]
Tested and working!
image

File: server/sv_dispatchcodes.lua [hunk 1]
```diff
@@ -39,5 +42,4 @@ dispatchCodes = {
 	["drugboatrobbery"] =  {displayCode = '10-31', description = "Suspicious Activity On Boat", radius = 0, recipientList = {'police'}, blipSprite = 427, blipColour = 26, blipScale = 1.5, blipLength = 2, sound = "robberysound"},
 	["unionrobbery"] =  {displayCode = '10-90', description = "Union Depository Robbery In Progress", radius = 0, recipientList = {'police'}, blipSprite = 500, blipColour = 60, blipScale = 1.5, blipLength = 2, sound = "robberysound"},
 	["carboosting"] =  {displayCode = '10-50', description = "Car Boosting In Progress", radius = 0, recipientList = {'police'}, blipSprite = 595, blipColour = 60, blipScale = 1.5, blipLength = 2, sound = "Lose_1st", sound2 = "GTAO_FM_Events_Soundset"},
-	["carjack"] =  {displayCode = '10-35', description = "Car Theft In Progress", radius = 0, recipientList = {'police'}, blipSprite = 595, blipColour = 60, blipScale = 1.5, blipLength = 2, sound = "Lose_1st", sound2 = "GTAO_FM_Events_Soundset"},
```
Comment (MonkeyWhisper) [inline server/sv_dispatchcodes.lua]: Why did you remove the carjack?

**Review by MonkeyWhisper — DISMISSED** [review]
How come you removed carjack? All else looks good.

File: server/sv_dispatchcodes.lua (hunk 1 above)
Comment (NoobySloth) [reply server/sv_dispatchcodes.lua]: Hey, 

I moved it to line 33 instead as it was in the section of -- rainmads
Although so is carboosting so ideally could do with moving that but it was pure OCD, it still exists

**Comment by pushkart2** [comment]
Is this tested? screenshots of it working would be great. Its a bunch of changes.

**Comment by NoobySloth** [comment]
> Is this tested? screenshots of it working would be great. Its a bunch of changes.

Locales work, I haven’t bought yacht heist. I will buy it and test it as soon as I can but seen as it’s a copy of previous alerts I don’t see it being an issue. Will test when I can!

**Comment by pushkart2** [comment]
> 

You dont need yacht heist to test it, register a command and trigger it

**Comment by NoobySloth** [comment]
> You dont need yacht heist to test it, register a command and trigger it

Ah yes, smart man you are. I will run a test tomorrow and post screenshots. :)

File: server/sv_dispatchcodes.lua (hunk 1 above)
Comment (ItsANoBrainer) [reply server/sv_dispatchcodes.lua]: @MonkeyWhisper It was moved up in the list, it still exists.

Comment (MonkeyWhisper) [reply server/sv_dispatchcodes.lua]: > @MonkeyWhisper It was moved up in the list, it still exists.

Thank you! I didn't see that. Thanks Nooby.

**Comment by MonkeyWhisper** [comment]
> > You dont need yacht heist to test it, register a command and trigger it
> 
> Ah yes, smart man you are. I will run a test tomorrow and post screenshots. :)

Tested and all Gucci here?

**Comment by NoobySloth** [comment]
> > > You dont need yacht heist to test it, register a command and trigger it
> > 
> > 
> > Ah yes, smart man you are. I will run a test tomorrow and post screenshots. :)
> 
> Tested and all Gucci here?

Works, although the event sends to the player's location, but will only trigger when on a boat anyway so not really much of an issue I guess.

image
image

**Comment by MonkeyWhisper** [comment]
Sorry! This has conflict, can you please fix? We will merge after.
